app/code/Magento/PaypalGraphQl/etc/schema.graphqls

```diff
@@ -51,6 +51,7 @@ input PaymentMethodInput {
     payflow_link: PayflowLinkInput @doc(description:"Required input for PayPal Payflow Link and Payments Advanced payments")
     payflowpro: PayflowProInput @doc(description: "Required input type for PayPal Payflow Pro and Payment Pro payments")
     hosted_pro: HostedProInput @doc(description:"Required input for PayPal Hosted pro payments")
+    payflowpro_cc_vault: VaultTokenInput
```

Comment:
Do we expect an additional logic behind this attribute? I mean, a possibility to reuse token via GraphQl?

Reply:
Hi @rogyar 
I double checked and I see that this parameter doesn't influence on saving or not saving card data of payflowpro payment method for future
Perhaps this parameter needs for something other.

Also note please I did `is_active_payment_token_enabler: Boolean` not required in graphql schema because if this parameter will be required we need input it for guest customers too . 
I think it's redundant.
`is_active_payment_token_enabler` is  require only for logged customers.
 I checks it in observer class
Thanks